src/braket/experimental/autoqasm/autograph/pyct/parser.py

```diff
@@ -128,6 +130,21 @@ def dedent_block(code_string):
   return new_code
 
 
+def in_hybrid_job():
+  if "AMZN_BRAKET_INPUT_DIR" in os.environ:
```

Comment:
minor comment: since this is in `autograph` source, we'll have to eventually contribute this to the standalone `autograph` repo when it becomes available. In that case, we might want to consider one more layer of abstraction here so that we can keep "Braket" and "hybrid job" concepts out of `autograph` code.

Reply:
It also makes it easier to maintain and easier to write unit tests when source code is injected outside of autograph.

Reply:
The source code injecting is now separated from autograph!